src/components/application_manager/src/commands/hmi/close_popup_request.cc

```diff
@@ -44,7 +44,11 @@ ClosePopupRequest::~ClosePopupRequest() {}
 
 void ClosePopupRequest::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
-
+  if(!CheckAvailabilityHMIInterfaces(application_manager_,
```

Comment:
I suggest to add an `UIRequestImpl` class for all VR-related Requests and avoid copy-pasta check.